rkt/pods.go

```diff
@@ -799,8 +799,17 @@ func getChildPID(ppid int) (int, error) {
 	return -1, ErrChildNotReady{}
 }
 
-// getPID returns the pid of the pod.
-func (p *pod) getPID() (pid int, err error) {
+// getPID returns the pid of the rkt process that started the pod.
```

Comment:
Well, the rkt process execs nspawn or kvm so in a way it is the rkt process. But you're right, `stage1 process` is better.